spec/spec_helper.rb

```diff
@@ -2,21 +2,6 @@
 Dir[Insights::API::Common::Engine.root.join("spec/support/**/*.rb")].each { |f| require f }
 require 'webmock/rspec'
 
-# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```

Comment:
Is this file auto generated or hand edited? From the deleted comments it seemed to be auto generated.

Reply:
The original is auto generated. I just pulled out all the commented out code to clean up the file.